src/tests/Logary.Targets.Stackdriver.Tests/Program.fs

```diff
@@ -0,0 +1,78 @@
+﻿module Program
+
+open System
+open Expecto
+open Hopac
+open Hopac.Infixes
+open Logary.Utils.Chiron
+open Logary
+open Logary.Target
+open Logary.Targets
+open Logary.Internals
+open Google.Logging.V2
+open Logary.Message
+open Google.Logging.Type
+open Google.Protobuf.WellKnownTypes
+
+let emptyRuntime = RuntimeInfo.create "tests"
+
+let flush = Target.flush >> Job.Ignore >> run
+
+let env k =
+  match Environment.GetEnvironmentVariable k with
+  | null -> failwithf "couldn't load key %s" k
+  | v -> v
+
+let logName = env "STACKDRIVER_LOG"
```

Comment:
There's a 60-day free trial with $300 worth of credit, and after that there is a fairly generous free tier for most services, including the information for stackdriver here: https://cloud.google.com/stackdriver/pricing

What I don't know precisely is what happens at the end of the 60 day trial.